packages/devtools_app/lib/src/debugger/file_search.dart

```diff
@@ -67,7 +67,7 @@ class _FileSearchFieldState extends State<FileSearchField>
       context: context,
       searchFieldKey: fileSearchFieldKey,
       onTap: _onSelection,
-      autocompleteMatchTileHeight: autocompleteMatchTileHeight,
+      autoCompleteMatchTileHeight: autoCompleteMatchTileHeight,
```

Comment:
this size needs to be scaled with the font size.

Reply:
Should now be scaling, but as mentioned in other comment - how do I choose a larger font size to verify this scales nicely?